package.json

```diff
@@ -109,7 +111,7 @@
   "main": "build/electron/electron/index.js",
   "husky": {
     "hooks": {
-      "pre-commit": "npx tslint --fix 'src/**/*.ts' && npx git-clang-format && npx pretty-quick --staged --pattern '**/*.html'"
+      "pre-commit": "npm run lint:ts -- --fix && npx git-clang-format && npx pretty-quick --staged --pattern '**/*.html'"
```

Comment:
Can we make `clang-format` or `pretty-quick` into lint actions that check only changed lines?

Reply:
Maybe in a follow up PR? Let's get this one in first.